bindings/ruby/lib/opendal_ruby/io.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
```

Comment:
We need to include a license for this file as well.

Reply:
Good catch! Fixed.